routers/contactsRouter.js

```diff
@@ -0,0 +1,97 @@
+const express = require('express');
+const router = express.Router();
+const Joi = require('joi');
+const responseNormalizer = require('../normalizers/response-normalizer');
+const errorWrapper = require('../helpers/error-wrapper');
+const fnContacts = require('../contacts');
+
+router.get(
+  '/',
+  errorWrapper(async (req, res) => {
+    const list = await fnContacts.listContacts();
+    return res.status(200).send(responseNormalizer(list));
+  }),
+);
+
+router.get(
+  '/:contactId',
```

Comment:
do not write any logics in routes

you should create new layer controller

so move logic from routers to controllers